pages/post/[postId].js

```diff
@@ -55,6 +53,7 @@ export default function Post({ post, isDynamicLink }) {
     const router = useRouter()
     useEffect(() => {
         if (isDynamicLink) router.push(`/post/${post.id}`, undefined, { shallow: true })
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The addition of `// eslint-disable-next-line react-hooks/exhaustive-deps` disables the lint warning for missing dependencies in the useEffect hook. This can lead to unexpected behavior if the dependencies of this effect change over time. If you are sure that the effect should only run when `isDynamicLink` changes, it's fine to leave it as is. But if there might be other dependencies in the future, consider removing this line to allow ESLint to warn about them.
```diff
     useEffect(() => {
         if (isDynamicLink) router.push(`/post/${post.id}`, undefined, { shallow: true })
-    // eslint-disable-next-line react-hooks/exhaustive-deps
     }, [isDynamicLink])
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->